src/Service/Services/OpenApiDocumentor.cs

```diff
@@ -0,0 +1,669 @@
+                        Format = formatMetadata
+                    });
+                }
+            }
+
+            OpenApiSchema schema = new()
+            {
+                Type = "object",
+                Properties = properties
+            };
+
+            return schema;
+        }
+
+        /// <summary>
+        /// Creates the default collection of responses for all requests in the OpenAPI
+        /// description document.
+        /// The OpenApiResponses dictionary key represents the integer value of the HttpStatusCode,
+        /// which is returned when using Enum.ToString("D").
+        /// The "D" format specified "displays the enumeration entry as an integer value in the shortest representation possible."
+        /// </summary>
+        /// <seealso cref="https://learn.microsoft.com/en-us/dotnet/standard/base-types/enumeration-format-strings#d-or-d"/>
+        /// <returns>Collection of default responses (400, 401, 403, 404).</returns>
+        private static OpenApiResponses CreateDefaultOpenApiResponses()
+        {
+            OpenApiResponses defaultResponses = new()
+            {
+                { HttpStatusCode.BadRequest.ToString("D"), CreateOpenApiResponse(description: nameof(HttpStatusCode.BadRequest)) },
+                { HttpStatusCode.Unauthorized.ToString("D"), CreateOpenApiResponse(description: nameof(HttpStatusCode.Unauthorized)) },
+                { HttpStatusCode.Forbidden.ToString("D"), CreateOpenApiResponse(description: nameof(HttpStatusCode.Forbidden)) },
+                { HttpStatusCode.NotFound.ToString("D"), CreateOpenApiResponse(description: nameof(HttpStatusCode.NotFound)) }
+            };
+
+            return defaultResponses;
+        }
+
+        /// <summary>
+        /// Converts the CLR type to System.Text.Json's JsonValueKind
+        /// to meet the data type requirement set by the OpenAPI specification.
+        /// The value returned is formatted for the OpenAPI spec "type" property.
+        /// </summary>
+        /// <param name="type">CLR type</param>
+        /// <seealso cref="https://spec.openapis.org/oas/v3.0.1#data-types"/>
+        /// <returns>Formatted JSON type name in lower case: e.g. number, string, boolean, etc.</returns>
+        private static string SystemTypeToJsonValueKind(Type type)
+        {
+            JsonValueKind openApiTypeName = type.Name switch
+            {
+                "String" => JsonValueKind.String,
+                "Guid" => JsonValueKind.String,
+                "Byte" => JsonValueKind.String,
+                "Int16" => JsonValueKind.Number,
+                "Int32" => JsonValueKind.Number,
+                "Int64" => JsonValueKind.Number,
+                "Single" => JsonValueKind.Number,
+                "Double" => JsonValueKind.Number,
+                "Decimal" => JsonValueKind.Number,
+                "Float" => JsonValueKind.Number,
+                "Boolean" => JsonValueKind.True,
```

Comment:
Need to identify workaround for having just "Boolean" but JsonValueKind.True and JsonValueKind.False are all that exist.

Reply:
fixed.